src/main/java/uk/gov/hmcts/probate/transformer/CallbackResponseTransformer.java

```diff
@@ -438,6 +438,14 @@ public CallbackResponse resolveStop(CallbackRequest callbackRequest) {
         return transformResponse(responseCaseDataBuilder.build());
     }
 
+
+    public CallbackResponse resolveCaseWorkerEscalationState(CallbackRequest callbackRequest) {
```

Comment:
maybe rename to set so it's clear this is a set not a resolve only

Reply:
updated